docs/Artifacts definition format and style guide.asciidoc

```diff
@@ -72,6 +74,12 @@ An object of digital archaeological interest.
 Where digital archaeology roughly refers to computer forensics without the
 forensic (legal) context.
 
+=== [[knowledge_base]]Knowledge Base
```

Comment:
The Knowledge Base is a GRR implementation detail, not part of the specification. One could use runtime environment variables to expands paths.

Reply:
I used it as an easy way to describe `provides` and parameter expansion. If that should not be part of the spec those parts have to be defined on another way.

Reply:
> If that should not be part of the spec those parts have to be defined on another way.

that needs to be assessed, currently provides is concept introduced by GRR.